games-puzzle/freesweep/freesweep-1.0.1.ebuild

```diff
@@ -17,6 +17,10 @@ DEPEND="${RDEPEND}
 	virtual/pkgconfig
```

Comment:
`virtual/pkgconfig` goes in `BDEPEND` under EAPI 7.

Reply:
updated.